manifests/init.pp

```diff
@@ -0,0 +1,166 @@
+# Sets up rkhunter to be run once per day
+#
+# @param minute_upd   Cron minute for update check
+# @param hour_upd     Cron hour for update check
+# @param monthday_upd Cron monthday for update check
+# @param month_upd    Cron month for update check
+# @param weekday_upd  Cron weekday for update check
+# @param minute       Cron minute
+# @param hour         Cron hour
+# @param monthday     Cron monthday
+# @param month        Cron month
+# @param weekday      Cron weekday
+#
+# @param package_ensure                The ensure status of packages to be managed
+#
+# @param rkhunter_conf_file            The path to the conf file
+#
+# @param rkhunter_conf_file_template   Template to use for the conf file
+#
+# @param check_for_updates             Check internet for definition updates
+#
+# @author https://github.com/simp/pupmod-simp-rkhunter/graphs/contributors
+#
+class rkhunter (
+  String[1]         $minute_upd                      = '0',
+  String[1]         $hour_upd                        = '0',
+  String[1]         $monthday_upd                    = '*',
+  String[1]         $month_upd                       = '*',
+  String[1]         $weekday_upd                     = '*',
+  String[1]         $minute                          = '10',
+  String[1]         $hour                            = '0',
+  String[1]         $monthday                        = '*',
+  String[1]         $month                           = '*',
+  String[1]         $weekday                         = '*',
+  String[1]         $package_ensure                  = 'latest',
+  String[1]  $rkhunter_conf_file                    = '/etc/rkhunter.conf',
+  String[1]  $rkhunter_conf_file_template           = 'rkhunter/rkhunter-conf.erb',
+  Optional[Boolean]    $check_for_updates            = undef,
+############start of rkhunter.conf options
+  Optional[Array[String[1]]]  $allowdevfile                      = undef,
+  Optional[Array[String[1]]]  $allowhiddendir                    = undef,
+  Optional[Array[String[1]]]  $allowhiddenfile                   = undef,
+  Optional[Array[String[1]]]  $allowipcpid                       = undef,
+  Optional[Array[String[1]]]  $allowipcproc                      = undef,
+  Optional[Array[String[1]]]  $allowipcuser                      = undef,
+  Optional[Array[String[1]]]  $allowprocdelfile                  = undef,
+  Optional[Array[String[1]]]  $allowproclisten                   = undef,
+  Optional[Array[String[1]]]  $allowpromiscif                    = undef,
+  Integer[0,2]                $allow_ssh_prot_v1                 = 0,
+  Enum['no','yes','unset']    $allow_ssh_root_user               = 'no',
+  Integer[0,1]                $allow_syslog_remote_logging       = 0,
+  Integer[0,1]                $append_log                        = 0,
```

Comment:
That goes against the default of the program, but I do agree that it should append, otherwise it keeps at most the past 2 runs, as it will make a `rkhunter.log` and a `rkhunter.log.old` when it runs and there is already one there; which means we'd be losing a number of days of runs.

As for logrotate, the logs are not that large, and on my test simp system the logs were getting rotated weekly into a `.gz` dated file.

Reply:
Do we need an explicit log rotate done for the module?